tests/drivers/dma/mux_req_gen/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
I think this would be better as a sample

Reply:
changed but this application is too specific to commit